Vilnius 2019 blog post announcing schedule and events and activities (#919)
writethedocs/www: docs/conf/vilnius/2019/news/announcing-schedule-activities.rst

Announcing Full Schedule, Tickets, Events, and Activities
=========================================================

Hello everyone!

Our first-year event in Vilnius is just over a week away, and we hope everyone is as excited as we are. We're writing today to announce the full presentation schedule and to give you the latest updates to help you make the most out of your conference experience.

Full presentation schedule
--------------------------

The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/vilnius/2019/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference.

We'll also be running the `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ on Sunday and `lightning talks <https://www.writethedocs.org/conf/vilnius/2019/lightning-talks/>`_ after lunch on Monday.

You'll be able to sign up for Writing Day projects and lightning talks during the conference.

Tickets are going!
------------------

We still have some tickets available, so head over to the `Tickets <https://www.writethedocs.org/conf/vilnius/2019/tickets/>`_ page and grab one!

> Due to catering and printing requirements, we will freeze ticket sales on **Monday, 27th May**. If you register after this date, we will not be able to print a badge for you (although you probably don't need to worry about the food!).

Reviewer: I take it this means you can't buy a ticket anymore, but you can still register it to your name, after this date? Or not? I am confused by the combination of "freeze sales" with "register after this date".

> Due to catering and printing requirements, we will freeze ticket sales on **Monday, 27th May**. If you register after this date, we will not be able to print a badge for you (although you probably don't need to worry about the food!).

Author: ```suggestion
Keep in mind that due to printing schedules, if you register after **Monday, 27th May** we will not be able to print a badge for you.
```

Events and activities
---------------------

Write the Docs is more than a conference. At each of our events, we try to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge.

Here is the overview of what you can expect during the conference:

> * **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. Please email us if you want to submit a project for the Writing Day!

Reviewer: "please email us" sounds contradictory with the earlier "you can sign up at the conference" ?

> * **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. Please email us if you want to submit a project for the Writing Day!

Author: ```suggestion
* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. You can sign up for a project during the day, or email us in advance if you want to have your project listed on our website.
```

> * **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *9:30am*. Please email us if you want to submit a project for the Writing Day!

Author: ```suggestion
* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. Please email us if you want to submit a project for the Writing Day!
```
